library/core/src/pin.rs

```diff
@@ -1292,7 +1292,7 @@ impl<Ptr: Deref> Pin<Ptr> {
     ///     // data that we pinned above, which we could use to move it as we have
     ///     // seen in the previous example. We have violated the pinning API contract.
     ///  }
-    ///  ```
+    /// ```
```

Comment:
If fixing this, let's also fix the previous line which is equally misaligned.

<img src="https://github.com/user-attachments/assets/b6bd9431-e6ff-4c48-8229-617ef14413ae" width="200">